### PR TITLE
Minor typo fixes

### PR DESCRIPTION
**Pull request by Akasurde**
Fixes https://pagure.io/freeipa/issue/6865

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

### CODE REVIEW

File: install/tools/man/ipa-adtrust-install.1 [hunk 1]
```diff
@@ -76,7 +76,7 @@ change the name. Please note that changing the NetBIOS name might break
 existing trust relationships to other domains.
 .TP
 \fB\-\-add\-sids\fR
-Add SIDs to existing users and groups as on of final steps of the
+Add SIDs to existing users and groups as one of the final steps of
```
Comment (stlaz) [inline install/tools/man/ipa-adtrust-install.1]: The "the" should have stayed there, it's related to the "run" rather than ipa-adtrust-install.

Comment (Akasurde) [inline install/tools/man/ipa-adtrust-install.1]: ok

Comment (Akasurde) [inline install/tools/man/ipa-adtrust-install.1]: Done.

File: install/tools/man/ipa-adtrust-install.1 [hunk 2]
```diff
@@ -76,7 +76,7 @@ change the name. Please note that changing the NetBIOS name might break
 existing trust relationships to other domains.
 .TP
 \fB\-\-add\-sids\fR
-Add SIDs to existing users and groups as on of final steps of the
+Add SIDs to existing users and groups as one of final steps of the
```
Comment (stlaz) [inline install/tools/man/ipa-adtrust-install.1]:
```diff
-Add SIDs to existing users and groups as on of final steps of the
+Add SIDs to existing users and groups as one of the final steps of the
```

**Comment by stlaz** [comment]
Thanks, ACK.

**Comment by tomaskrizek** [comment]
master:

* cb869314729022d4c16339bb08b79c5c30fe29df Minor typo fixes

**Comment by Akasurde** [comment]
@stlaz @tomaskrizek Thanks.

**Pull request closed without merging**